app/api/routes/status.py

```diff
@@ -1,8 +1,77 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, HTTPException, Depends, Header
+from uuid import UUID, uuid4
+from typing import Dict
+from pydantic import BaseModel, Field
+from models import StatusCreate,StatusUpdate,Status
+from dependencies import get_current_admin,get_current_time,statuses,orders_db
 
 router = APIRouter()
 
-# make sure to remove this function
-@router.get("/example")
-def example():
-    return {"message": "this is an example"}
\ No newline at end of file
+"""Create Status"""
+@router.post("/statuses/", status_code=201)
+async def create_status(status: StatusCreate, admin: bool = Depends(get_current_admin)):
+# Check if the status is unique or not
+    try:
+        if status.name.lower() in statuses:
+            raise HTTPException(status_code=400, detail="Status name must be unique.")
+        
+        new_status = Status(name=status.name)
+        statuses[new_status.id] = new_status
+        return new_status.to_dict()
+    
+    except HTTPException as e:
+        raise e(status_code=400, detail="Bad Request")
+    except Exception as e:
+        raise HTTPException(status_code=500, detail="Internal Server Error")
+"""Get the status by the ID"""
+@router.get("/statuses/{status_id}",status_code=200)
+async def get_user(status_id: UUID,admin: bool = Depends(get_current_admin)):
+    try:
+        status = statuses.get(status_id)
+        if status_id not in statuses:
```

Comment:
Since in the previous line you get the status, there are two suggestions here
1. Move the previous line after the condition 
2. Or here check if status is empty/null

Reply:
Like line 46

Reply:
Done